Source/Rows/Common/FieldRow.swift

```diff
@@ -86,8 +86,25 @@ open class FieldRow<Cell: CellType>: FormatteableRow<Cell>, FieldRowConformance,
     open var keyboardReturnType: KeyboardReturnTypeConfiguration?
 
     /// The percentage of the cell that should be occupied by the textField
-    open var textFieldPercentage: CGFloat?
-
+	@available (*, deprecated, message: "Use titleLabelPercentage instead")
+	open var textFieldPercentage : CGFloat? {
+		get {
+			guard let titlePercentage = titlePercentage else { return nil }
+			return 1 - titlePercentage
```

Comment:
Given that `titlePercentage`  is a `CGFloat`, I do not see how we could use `map` on it. Am I missing something here?

If you would like to opt for a more concise style, I could offer the following one-liner:

```
return titlePercentage != nil ? 1 - titlePercentage! : nil
```